.github/workflows/samples.yml

```diff
@@ -21,9 +22,12 @@ jobs:
           - slack-samples/deno-message-translator
           - slack-samples/deno-request-time-off
           - slack-samples/deno-simple-survey
-        # we test on both most recent stable version of deno (v1.x) as well as
+        # we test on both most recent stable version of deno (v1.x, v2.x) as well as
         # the version of deno used by Run on Slack (as noted in https://api.slack.com/slackcli/metadata.json)
-        deno-version: [v1.x, v1.45.4]
+        deno-version:
+          - v1.x
+          - v1.45.4
```

Comment:
This might've been updated recently, but I'm finding `1.46.2` is the current ROSI runtime?

https://api.slack.com/slackcli/metadata.json